ec/src/models/short_weierstrass/affine.rs

```diff
@@ -344,38 +343,18 @@ impl<P: SWCurveConfig> From<Projective<P>> for Affine<P> {
 }
 
 impl<P: SWCurveConfig> CanonicalSerialize for Affine<P> {
-    #[allow(unused_qualifications)]
```

Comment:
I'm not sure what this does, but there are no lint errors after removing it. Let me know if it's wrong to remove

Reply:
I dug into the history and it's been here for at least three years, I'm guessing it was kept long after no longer being needed

Reply:
Yeah, we can remove it if it doesn't trigger any linter errors.